client/packages/system/src/Program/utils.ts

```diff
@@ -0,0 +1,17 @@
+import { FilterOptionsState, RegexUtils } from '@openmsupply-client/common';
+import { ProgramDocumentFragment } from './api';
+
+export interface ProgramSearchProps {
+  disabledPrograms?: string[];
+  open: boolean;
+  onClose: () => void;
+  onChange: (name: ProgramDocumentFragment) => void;
+}
```

Comment:
ah yeah. copy paste that one. I left it like that, since it was done this way elsewhere. 
Have changed though, I expect the interface to be with the component.